bin/mastarm-deploy

```diff
@@ -120,8 +194,24 @@ function logToMsTeams (error) {
       }
     ]
   }]
+  if (configCommit && configRemoteUrl) {
+    potentialAction.push({
+      '@type': 'OpenUri',
+      name: `View Config Commit on Github`,
+      targets: [
+        {
+          os: 'default',
+          uri: `${configRemoteUrl}/tree/${configCommit}/${configDir}`
+        }
+      ]
+    })
+  }
   const text = `📄 *commit:* ${pkg.name}@${commit.slice(0, 6)}\n
   👤 *deployed by:* ${username.sync()}\n
+  ${configCommit
+    ? `🎛️ *config:* configurations@${configCommit.slice(0, 6)}\n
+  📂 *config folder:* ${configDir}\n` // improper indenting here needed to properly format on MS Teams
+    : '🎛️ *config:* unknown configuration data!\n'}
   🚦 *mastarm:* v${mastarmVersion}\n
   ☁️ *cloudfront:* ${cloudfront}\n
   🌱 *env:* ${env}\n
```

Comment:
One thing I didn't notice in the previous PR (#284) was that the items in this list are italicized and not bolded. I think it's easier to read/distinguish keys from values if they're bolded instead.